File: dataset/codenet/Java/Code/s511234019.java
/* package whatever; // don't place package name! */

import java.util.*;
import java.lang.*;
import java.io.*;

/* Name of the class has to be "Main" only if the class is public. */
public class Main
{
	public static void main (String[] args) throws java.lang.Exception
	{
		Scanner in =new Scanner(System.in);
		int a,b;
		a=in.nextInt();
		b=in.nextInt();
		if(a%2==0 || b%2==0 )
			System.out.println("No");
		else
			System.out.println("Yes");
	}
}

File: output/llama_4_maverick/codenet/algo_based_translation/java/python/s511234019.py

import sys

class Main:
    def main():
        try:
            a = int(sys.stdin.readline().strip())
            b = int(sys.stdin.readline().strip())
            if a % 2 == 0 or b % 2 == 0:
                print("No")
            else:
                print("Yes")
        except Exception as e:
            raise Exception(e)

if __name__ == "__main__":
    Main.main()

